dirwatcher.js

```diff
@@ -8,16 +8,16 @@ class DirWatcher {
 	}
 
 	watch(path, delay) {
-		let fsTimeout;
+		let fsTimeouts = [];
```

Comment:
It should be an object, not an array.

Reply:
yep, thanks, initially was another idea)